src/components/chips/MovingTypeChips.tsx

```diff
@@ -41,7 +41,7 @@ export default function MovingTypeChips({ type }: MovingTypeChipsProps) {
           </div>
         </div>
       )}
-      {type === 'APPOINTMENT' && (
+      {(type === 'APPOINTMENT' || specificRequest === true) && (
```

Comment:
이 구조를 추가하신 이유가 뭔가요??

Reply:
이사 유형에서 데이터로 type은 small, home, office만 보내주고 지정견적요청의 유무는 specificRequest boolean 값으로 보내주고 있기 때문에 위 해당하는 ui를 사용하기 위해 추가했습니다. 아예 변경하려고 했으나 기존의 코드에서 저 컴포넌트를 사용하는 곳에서 에러가 나올 것 같아 일단 추가했습니다

Reply:
사용하시는 페이지에서 
```
{specificRequest && <MovingTypeChips type="APPOINTMENT" />}
```
형식으로 사용하기는 어려운가요??

Reply:
음 type으로 APPOINTMENT가 안와서 사용이 안되요 근데 궁금한게 type으로 APPOINTMENT를 받는 데이터가 있나요??

Reply:
지정견적요청 유무는 백에서 boolean 값으로 주는데 저 APPOINTMENT는 언제 받는건가요??

Reply:
APPOINTMENT는 그냥 프론트에서 작업을 진행할 때 임시적으로 지정해뒀던 값인데, 지정견적만 boolean으로 빼서 로직을 짜는 것보단 type으로 일관되게 주는 게 더 좋을 것 같아서 수정하지 않았습니다!
```
{specificRequest && <MovingTypeChips type="APPOINTMENT" />}
```
이렇게 코드를 짜면 `specificRequest`가 `true` 일 때 `<MovingTypeChips type="APPOINTMENT" />` ui를 보여주게 되는 것이니 저렇게 추가하는 것보다 위의 코드처럼 사용해주시는 게 더 좋을 것 같다는 생각이 듭니다!

Reply:
음 type으로 일관되게 주신다는 목적은 이해가 되지만 서버에서 받은 데이터로 type에 따라 구분하는 컴포넌트에서 사용되지 않는 type을 남겨 사용하는 것 보다는 small, home , office 제외의 타입은 제거하는 것이 더 좋을 것 같습니다! 그리고 사용하는 곳이 많은 컴포넌트인데 위처럼 사용하는 곳에서 모두 선언해주는 것 보다는 컴포넌트에 데이터 넘겨주고 그 데이터에 따라서 구분되는 것이 더 좋을 것 같습니다!

Reply:
waiting과 received도 삭제하자는 말씀이신가요?
waiting과 received는 이미 사용처가 있습니다

Reply:
아니요 waiting과 received를 type으로 안받자나요 waiting received를 type으로 백에서 데이터를 보내주나요?

Reply:
MovingTypeChipsProps에 확인해보시면 type에 waiting과 received도 있습니다!
해당 type들도 백엔드에서 보내주지 않지만 재원님이 일관성을 위해 저렇게 프로그래밍 하신 것 같습니다.

Reply:
프론트에서의 type이 아니라 재원님이 api 연결하시면 waiting과 received 데이터를 type이 아닌 status나 progress로 받을텐데 type으로는 small, home, office들의 타입을 받아서 MovingChips 컴포넌트에서 type에 대한 ui를 그려줘야 하고, waiting과 received에 대한 ui를 그릴려면 효빈님 말씀대로 호출할 때 마다 status 혹인 progress가 status === 'waiting'식으로 조건 달아서 보여줘야 할텐데 그것 보다는 컴포넌트 내부에 사용하지 않는 타입은 제거하고 받아오는 데이터로 정의하는게 좋을 것 같다고 말씀드린겁니다!

Reply:
말씀하시는 type이 serviceType인가요?

Reply:
넵 맞습니다

Reply:
흠 정하기 나름인 문제이긴 하지만 원본 코드를 훼손하지 않는다면 어느 방면이든 상관 없을 듯 합니다.
각자 편한 방식으로 사용하면 될 것 같아요!
약간의 코드 수정을 요청드리자면 
```suggestion
      {(type === 'APPOINTMENT' || specificRequest) && (
```
이렇게 하시면 더 깔끔해보일 것 같습니다!

Reply:
넵 알겠습니다! true를 빼는 것이 더 간결해서 좋아보기는 합니다만 예를 들어 문자열이나 [] 같은 truthy값은 모두 해당이 되서 조건이 참이 될텐데 위 처럼 수정해도 괜찮을까요??

Reply:
저는 지정견적이 true인 경우에만 specificRequest라는 prop을 사용할 것 같아서 상관 없을 것 같다고 판단하였습니다!

Reply:
음 일단 알겠습니다 수정했습니다!